kratos/tests/cpp_tests/utilities/test_variable_utils.cpp

```diff
@@ -0,0 +1,154 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Ruben Zorrilla
+//
+
```

Comment:
Strange this wasn't testes

Reply:
All tests are in Python. I think this one has a pretty good coverage.